src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -12,20 +12,21 @@ namespace Microsoft.ML
     /// <summary>
     /// Specifies input and output column names for a transformation.
     /// </summary>
-    public sealed class ColumnOptions
+    [BestFriend]
+    internal sealed class ColumnOptions
```

Comment:
>ColumnOptions [](start = 26, length = 13)

So this is interesting. You're hiding *all* multi-column mappings. I guess that's probably for the best till now till we figure out the story around here.